atomic_reactor/koji_util.py

```diff
@@ -88,9 +120,7 @@ def koji_login(session,
 
         result = session.krb_login(**kwargs)
 
-    if not result:
-        raise RuntimeError('Unable to perform Koji authentication')
-
+    # koji.ClientSession.*_login raise errors or return True, so do the same here.
```

Comment:
Why this change?

I think it's pretty bad that koji's lib doesn't raise an error if auth fails. This was a way to make its usage a little bit nicer. With this change, all callers must be modified to verify that auth actually succeeded.

Reply:
Doh! I missed that *_login can return False under some circumstances. I'll fix the code and the test case.

Reply:
Fixed.